test/middleware.js

```diff
@@ -2,6 +2,9 @@ const express = require('express');
 
 function middleware() {
   const api = express();
+  api.get('/weather/taipei', (req, res) => {
+    res.send(101);
```

Comment:
101 is a clever number, but "switching protocols" will result in some strange behavior.

probably 100 Continue is best, but you will need a POST endpoint to handle this.